gradle/dokka-multiplatform-example/build.gradle.kts

```diff
@@ -38,41 +37,20 @@ kotlin {
     }
 }
 
-tasks {
-    val dokka by getting(DokkaTask::class) {
-        outputDirectory = "$buildDir/dokka"
-        outputFormat = "html"
-
-        multiplatform {
-            val js by creating {} // this platform uses autoconfiguration. Sources are fetched from kotlin-multiplatform plugin
-
-            register("customName") { // or `val customName by creating {}`. The name is different than `jvm` in Kotlin plugin, so
-                                           // sourceRoots, classpath, platform and targets must be passed explicitly
-                targets = listOf("JVM")
-                platform = "jvm"
-                sourceRoot {
-                    path = kotlin.sourceSets.getByName("jvmMain").kotlin.srcDirs.first().toString()
-                }
-                sourceRoot {
-                    path = kotlin.sourceSets.getByName("commonMain").kotlin.srcDirs.first().toString()
-                }
+tasks.withType<DokkaTask> {
+    dokkaSourceSets {
+        register("commonMain") {
+            sourceRoot { path = "src/commonMain/kotlin" }
+        }
```

Comment:
I'd personally leave the `customName` source sets here, they were created explicitly to show how to configure a pass manually if somebody has some weird build (or doesn't have the Kotlin plugin for some reason)